bindings/python/bergamot.cpp

```diff
@@ -0,0 +1,227 @@
+#include <pybind11/iostream.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+#include <translator/annotation.h>
+#include <translator/parser.h>
+#include <translator/project_version.h>
+#include <translator/response.h>
+#include <translator/response_options.h>
+#include <translator/service.h>
+#include <translator/translation_model.h>
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace py = pybind11;
+
+using marian::bergamot::AnnotatedText;
+using marian::bergamot::ByteRange;
+using marian::bergamot::ConcatStrategy;
+using marian::bergamot::Response;
+using marian::bergamot::ResponseOptions;
+using Service = marian::bergamot::AsyncService;
+using _Model = marian::bergamot::TranslationModel;
+using Model = std::shared_ptr<_Model>;
+using Alignment = std::vector<std::vector<float>>;
+using Alignments = std::vector<Alignment>;
+
+PYBIND11_MAKE_OPAQUE(std::vector<Response>);
+PYBIND11_MAKE_OPAQUE(std::vector<std::string>);
+PYBIND11_MAKE_OPAQUE(Alignments);
+
+class ServicePyAdapter {
+public:
+  ServicePyAdapter(const Service::Config &config)
+      : service_(make_service(config)) {}
+
+  std::shared_ptr<_Model> modelFromConfig(const std::string &config) {
```

Comment:
From my understanding, this file should be a very thin python interface that calls bergamot functions. It shouldn't duplicate functionality from other parts of the source code.

Reply:
> very thin python interface

This is the transition boundary into python. Rules move from efficiency to [python zen](https://www.python.org/dev/peps/pep-0020/#id2) here. Sometimes I want to tweak config operating YAML, sometimes I like just supplying the config path. As of now, there's a coupling with Service that we want to get rid of. These functions are intended to be deprecated in favor of `TranslationModel` constructors following #257, until then these are here to stay.